common/traits/wc_genetic_traits.txt

```diff
@@ -224,4 +226,41 @@ magic_good_noninheritable = {
 			desc = trait_magic_good_noninheritable_character_desc
 		}
 	}
+}
+
+dire = {
+	opposites = {
+		weak
+	}
+
+	index = 2508
+
+	prowess = 10
+	dread_baseline_add = 15
+	
+	health = -0.05
+
+	diplomacy = -1
+	stewardship = -1
+	intrigue = -1
+	learning = -1
```

Comment:
I created a balancing card here. You can balance the modifiers as you want.
https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=1768507286&range=A3:A6